### DISCUSSION TITLE
platform/actors/running/input-and-output

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#903] [Comments]
# platform/actors/running/input-and-output

Configure your Actor's input parameters using Apify Console, locally or via API. Access parameters in key-value stores from your Actor's code.

https://docs.apify.com/platform/actors/running/input-and-output

**Comment by emilygrove** [comment]
I have an Apollo account with 12,000 credits used. Now I have extracted 3000 more records using APIFY.I want to ask you that last time I tried to withdraw 3727 records they said your link is broken your actor is broken I want to know if this is my paid account if I bought it So can I pull the record from Apollo, or is it telling me to change the link, but I can't change the link?I want to know from you that if I purchase APIFY account, will I be able to extract data easily. One thing I want to mention is that I have a paid account with Apollo but no credit in it.Don't you have to tell me again and again that your actor is break or change the link of Apollo again? Tell me how to change the link of Apollo .I have only one Apollo account. Will it be beneficial for me?
